### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -10,6 +10,9 @@ set(Brion_VERSION_ABI 9)
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
   ${CMAKE_SOURCE_DIR}/CMake/common ${CMAKE_SOURCE_DIR}/Pydoxine/CMake)
+if(NOT EXISTS ${CMAKE_SOURCE_DIR}/CMake/common/Common.cmake)
+  message(FATAL_ERROR "CMake/common missing, run: git submodule update --init")
+endif()
 
 include(GitExternal)
```
Comment (tribal-tec) [inline CMakeLists.txt]: not needed anymore

Comment (hernando) [inline CMakeLists.txt]: Actually it is, but Pydoxine. Unless we make this also a submodule, but then it'd be replicated by RTNeuron as we discussed.

Comment (tribal-tec) [inline CMakeLists.txt]: Ah, sure, my bad.

Comment (tribal-tec) [inline CMakeLists.txt]: But CMake/common is not removed yet from .gitexternals?

Comment (hernando) [inline CMakeLists.txt]: That I forget to do it.

**Pull request merged**